planning/behavior_path_planner/src/scene_module/avoidance/avoidance_module.cpp

```diff
@@ -497,6 +497,14 @@ AvoidPointArray AvoidanceModule::calcRawShiftPointsFromObjects(
     const auto shift_length = isOnRight(o)
                                 ? std::min(o.overhang_dist + avoid_margin, getLeftShiftBound())
                                 : std::max(o.overhang_dist - avoid_margin, getRightShiftBound());
+
+    if (
+      std::fabs(shift_length) <
+      std::fabs(o.overhang_dist) - vehicle_width - lat_collision_safety_buffer) {
```

Comment:
@zulfaqar-azmi-t4 
I think this should be a `lat_collision_safety_margin`  because safety buffer is optional margin to avoid

Reply:
i think `lat_collision_safety_buffer` can be reuse for this task. Since we actually will create the optional margin to ignore the shift. 

But i think the name is quite confusing and wonder if we should just change the parameter`lateral_collision_safety_margin` to `additional_safety_margin` instead.